### PR TITLE
Feature/vardict_recipe_single

### PR DESCRIPTION
**Pull request by hassanfa**
Added another subroutine to handle single mode analysis to address issue #376 

The new subroutine is very similar to paired analysis mode. I have made it intentionally for possible expansions in the future to handle multiple normal tissue types for paired analysis.

### CODE REVIEW

File: lib/MIP/Recipes/Analysis/Vardict.pm [hunk 1]
```diff
@@ -476,6 +476,431 @@ sub analysis_vardict_tn {
 }
 
 sub analysis_vardict_single {
+    ## Function : Analysis recipe for varDict variant calling
```
Comment (henrikstranneheim) [inline lib/MIP/Recipes/Analysis/Vardict.pm]: Add new line and fix indent

File: lib/MIP/Recipes/Analysis/Vardict.pm [hunk 2]
```diff
@@ -475,6 +475,435 @@ sub analysis_vardict_tn {
...
+
+    }
+
+    ## Prepare an array of inputs for vardict
+    my @infile_path =
+      map { $file_path_prefix{$_} . $UNDERSCORE . $infile_suffix }
+      @{ $active_parameter_href->{sample_ids} };
+
+    ## Vardict
+    ## Note: Vardict doesn't runs the analysis based on the input bed file, so if the input bed file points towards the
```
Comment (henrikstranneheim) [inline lib/MIP/Recipes/Analysis/Vardict.pm]: Move this to docs as discussed previously

**Comment by hassanfa** [comment]
@henrikstranneheim I addressed the comments.

**Comment by henrikstranneheim** [comment]
👍

**Pull request merged**